src/features/app/components/Loading/Loading.js

```diff
@@ -7,8 +7,8 @@ import {
   DoubleBounceWithDelay,
 } from './Loading.styles';
 
-const Loading = ({ styles, className }) => (
-  <Wrapper styles={styles} className={className}>
+const Loading = ({ className }) => (
```

Comment:
Do we really need the `className` prop? I mean we can override the styles with `@emotion`

Reply:
In order to override styles with emotion, using the `styled(Component)` syntax, that component must receive a `className` prop.